docs-src/dev-guide/official-sdks.md

```diff
@@ -19,6 +19,5 @@ Each Temporal SDK targets a specific programming language.
 - [Python SDK developer's guide](/python/introduction-to-python-sdk)
 - [TypeScript SDK developer's guide](/typescript/introduction-to-typescript-sdk)
 - [.NET SDK API reference](https://dotnet.temporal.io/)
-- [PHP SDK API reference](https://php.temporal.io/namespaces/temporal.html)
```

Comment:
@josh-berry Please confirm this.

Reply:
My bad! I somehow didn't see it when I added the link. Yes I'm good with this, sorry for the duplicate!